### DISCUSSION TITLE
Send ctrl+# to neovim

### DISCUSSION THREAD

**Discussion by BaldFabi** [microsoft/terminal#14077] [Q&A]
I currently try to get my shortcut `ctrl+#` working in neovim with Windows Terminal (using an iso keyboard).
After discovering [the issue](https://github.com/microsoft/terminal/issues/3799) to send escape sequences is closed,I thought something like this should work but I think I just don't understand the escape sequence correctly.
```json
    {
      "command": {
        "action": "sendInput",
        "input": "\u001b[\u0023"
      },
      "keys": "ctrl+#"
    }
```
Sending something with `ctrl+#` to wsl (like ordinary letters) works.
Whats the correct value for `input` to get this working?

**Comment by AndreJorgeLopes** [comment]
Hey mate, I figured out a workaround that works at least with zsh.
So, that keybinding does not send ^#, but instead sends ^[[#, so I worked around that by binding "\u001b[\u0023" to the command I wanted to execute on zsh. (The wt sendInput command is still needed)
Here you have an example of my keybindings on the .zshrc file:
```bash
bindkey -e
bindkey '\u001b[\u0023' kill-whole-line
```
